### PR TITLE
navbar + conditional rendering

### PR DESCRIPTION
**Pull request by xshi0603**

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (sdhani) [inline client/src/index.js:6]: Nicee 👌 

Are there any additional files that need to be committed to get the project running with `bootstrap` or a local setup required to use that distro of the package?

We could alternatively use `yarn add bootstrap` to add the package to the project if it's on the same distro as the one referenced here.

File: client/src/components/Navbar.jsx [hunk 2]
```diff
@@ -1,27 +1,43 @@
-import React from "react";
-import { useHistory } from "react-router-dom";
+import {Navbar as BootstrapNavbar, Nav} from 'react-bootstrap/'
+import React, { useContext } from "react";
+import { UserContext } from "../utils/auth";
 
 // TODO: Handle Navigation B/t components better
-export const Navbar = () => {
-  const history = useHistory();
+// TODO: add color changing on phone dropdowns
```
Comment (sdhani) [inline client/src/components/Navbar.jsx:6]: 👍

File: client/src/components/Navbar.jsx [hunk 3]
```diff
@@ -1,27 +1,43 @@
...
+        <Nav className="ml-auto">
+          {/* TODO: add a switch that only shows one */}
 
-  return (
-    <nav>
-      <button onClick={handleHome}> Home </button>
-      <button onClick={handleLogin}> Login </button>
-      <button onClick={handleRegister}> Register </button>
-    </nav>
+          {!userData ?
```
Comment (sdhani) [inline client/src/components/Navbar.jsx:24]: 👍

File: client/src/components/Navbar.jsx [hunk 4]
```diff
@@ -1,27 +1,43 @@
...
+export const Navbar = ({logoutHandler}) => {
+  const { userData } = useContext(UserContext);
 
-  const handleRegister = () => {
-    history.push("/register");
-  };
+  return (
+    <BootstrapNavbar fixed="top" expand="lg" className="navbar navbar-dark ">
+      <BootstrapNavbar.Brand href="/">Charity Finder</BootstrapNavbar.Brand>
+      <BootstrapNavbar.Toggle aria-controls="basic-navbar-nav" />
```
Comment (sdhani) [inline client/src/components/Navbar.jsx:15]: Nice use of adding accessibility features ✨

**Review by sdhani — CHANGES_REQUESTED** [review]
Looks good! 👌

Just need some verification on `bootstrap` setup options and we're good to go. ^^

File: client/src/index.js (hunk 1 above)
Comment (dliang2) [reply client/src/index.js:6]: You are right, updated the dependencies in the package and yarn lock files

File: client/src/components/Navbar.jsx [hunk 5]
```diff
@@ -1,27 +1,43 @@
-import React from "react";
-import { useHistory } from "react-router-dom";
+import {Navbar as BootstrapNavbar, Nav} from 'react-bootstrap/'
```
Comment (sdhani) [inline client/src/components/Navbar.jsx]: `import {Navbar as BootstrapNavbar, Nav} from 'react-bootstrap/'`, you can drop the `/` here

File: client/src/index.js (hunk 1 above)
Comment (sdhani) [inline client/src/index.js:6]: Do we still need this line?

**Review by sdhani — APPROVED** [review]
Awesome!! 🚀 

Added two more notes, feel free to merge after that 😊👌✨

File: client/src/index.js (hunk 1 above)
Comment (dliang2) [reply client/src/index.js:6]: yes, tried deleting the line and the styling got messed up without it

Comment (sdhani) [reply client/src/index.js:6]: Nvm, it looks like this line is needed 😅, nice job!

**Pull request merged**